nanofi/include/cxx/C2CallbackAgent.h

```diff
@@ -47,7 +47,10 @@ class C2CallbackAgent : public c2::C2Agent {
 
  public:
 
-  explicit C2CallbackAgent(const std::shared_ptr<core::controller::ControllerServiceProvider> &controller, const std::shared_ptr<state::StateMonitor> &updateSink, const std::shared_ptr<Configure> &configure);
+  explicit C2CallbackAgent(const std::shared_ptr<core::controller::ControllerServiceProvider> &controller,
```

Comment:
This class is not used anywhere in the code today. It will be removed later when nanofi c2 work gets completed.